engines/saga/introproc_ite.cpp

```diff
@@ -128,6 +128,8 @@ EventColumns *Scene::queueIntroDialogue(EventColumns *eventColumns, int n_dialog
 		textEntry.rect.top = INTRO_DE_CAPTION_Y;
 	} else if (_vm->getLanguage() == Common::IT_ITA) {
 		textEntry.rect.top = INTRO_IT_CAPTION_Y;
+	} else if (_vm->getLanguage() == Common::FR_FRA) {
+		textEntry.rect.top = INTRO_IT_CAPTION_Y;
```

Comment:
Wouldn't it be cleaner to add a INTRO_FR_CAPTION_Y define?

Reply:
#define INTRO_DE_CAPTION_Y 160
#define INTRO_IT_CAPTION_Y 160

DE and IT are already same, and it works perfectly like that in French.
But if you think it's better, you can also add a FR one with same setting.

#define INTRO_FR_CAPTION_Y 160